### PR TITLE
refactor(paths): update leaning paths to use new design system, fixes #6917

### LINKED ISSUE #6917: Rollout the design system on the Learn landing pages

### LINKED ISSUE #6955: Update design of leaning path collections
Use new design system for learning path pages, such as [Web Vitals](https://web.dev/learn-web-vitals/) or [Angular](https://web.dev/angular)

### PR DESCRIPTION
**Pull request by MichaelSolati**
Fixes #6917

Changes proposed in this pull request:

- Update `src/site/_includes/path.njk` to use new design system.
- Update `src/site/_includes/partials/topic.njk` to use new design system.
- Update `src/site/_includes/partials/item.njk` to use new design system.
- Add `src/scss/pages/_path.scss` for CSS specific to path pages (such as `/angular`), specifically the lines dividing the topics.

Here's a before:
image
https://web.dev/angular/

And an after:
image
https://deploy-preview-6956--web-dev-staging.netlify.app/angular/

When you're ready to submit your PR, don't forget to add the `$-presubmit` label.

### CODE REVIEW

**Comment by devnook** [comment]
Great start, Michael. Can you please style the new page so that it looks like the old one? We don't intend to change th code design on it. Let me know if you have questions.

**Comment by hankchizljaw** [comment]
Looking like a good start! Some visual feedback from me:

- [ ] In the hero section, it'll be better to use the sidebar, just like you did with the overview section, because the topic's graphic will align nicely with the TOC then 
- [ ] In the overview section, opt for `pad-block-size-1`, rather than `pad-size-1` because the latter adds padding to all sides, whereas the former only adds vertical padding
- [ ] It looks like in the hero section, the summary hasn't rendered the markdown, so `flow` isn't working
- [ ] Maybe try to add `text-size-6` to that main heading too 
- [ ] For the topics, I say we aim for more consistency with the courses section and render h2s and links, j[ust like on the forms course](https://web.dev/learn/forms/#form-controls-and-form-fields), as an example. The borders and blue headings just don't match the new design system's simplicity.
- [ ] The benefit of that is `prose` does the anchor link styles for free, too

File: src/scss/pages/_learn-paths.scss [hunk 1]
```diff
@@ -0,0 +1,34 @@
+@media (min-width: 865px) {
```
Comment (devnook) [inline src/scss/pages/_learn-paths.scss:1]: I think the template that is generating those pages is called path.njk. Let's keep the naming consistent, so rename s/_learn-paths.scss/_path.scss

Comment (devnook) [inline src/scss/pages/_learn-paths.scss:1]: It looks like this stylesheet replaces the old styles/pages/_path.scss. Please remove that stylesheet if not needed anymore.

File: src/site/_includes/partials/item.njk [hunk 2]
```diff
@@ -4,7 +4,6 @@
   {% if pathItem.url %}
     <li class="w-path-listitem">
```
Comment (devnook) [inline src/site/_includes/partials/item.njk:5]: "w-xxxx" classes are deprecated in the new design system

File: src/scss/pages/_learn-paths.scss [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  .learn__path-hero {
+    height: auto;
+    max-width: initial;
+    width: 420px;
+  }
+}
+
+.learn-header {
```
Comment (devnook) [inline src/scss/pages/_learn-paths.scss:9]: If you use BEM notation (.learn__path-hero) stay with BEM (so .learn-header and .learn-link are incorrect)

File: src/scss/pages/_learn-paths.scss [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+
+  @media (min-width: 865px) {
+    align-items: center;
+    display: flex;
+    justify-content: space-between;
+    text-align: initial;
+  }
+}
+
+.headline-link {
```
Comment (devnook) [inline src/scss/pages/_learn-paths.scss:21]: 1. We already have "headline" element in our design-system, so please don't reuse that name in other context.

**Review by devnook — CHANGES_REQUESTED** [review]
Reviewed

File: src/site/_includes/path.njk [hunk 5]
```diff
@@ -10,62 +10,66 @@ eleventyComputed:
...
+          </ul>
+        </div>
+      </div>
     </div>
-  </section>
+  </div>
+</div>
 
-  <hr>
+<div class="wrapper"><hr/></div>
```
[the remark this answers is not in the document]
Comment (MichaelSolati) [reply src/site/_includes/path.njk:69]: I think this was intended, I got this from feedback from @hankchizljaw, I'm fine with removing the wrapper though, I just would want to confirm that is the desired look.

File: src/site/_includes/partials/topic.njk [hunk 6]
```diff
@@ -1,23 +1,27 @@
...
-  </h2>
-  <ul class="w-path-list">
-    {% if topic.pathItems %}
-      {% set section = topic %}
-      {% include 'partials/item.njk' %}
-    {% endif %}
-  </ul>
+<div class="wrapper pad-block-size-{{"2" if loop.first else "1"}}">
+  <div class="prose">
+    <h2 id="{{ topic.title | i18n(locale) | slug }}" class="text-size-2 headline-link">
```
Comment (devnook) [inline src/site/_includes/partials/topic.njk:3]: I don't think headline-link class is defined anymore

File: src/site/_includes/path.njk (hunk 5 above)
[the remark this answers is not in the document]
Comment (devnook) [reply src/site/_includes/path.njk:71]: Can you point me to it? I cannot find it. or was it on chat?

**Review by devnook — COMMENTED** [review]
Only some small questions left!

File: src/site/_includes/path.njk (hunk 5 above)
[the remark this answers is not in the document]
Comment (MichaelSolati) [reply src/site/_includes/path.njk:71]: It was in chat, I just shared it with you in chat.

File: src/site/_includes/partials/topic.njk (hunk 6 above)
Comment (MichaelSolati) [reply src/site/_includes/partials/topic.njk:3]: Yup, it you're right. I know I had seen it before, I guess it was removed at some point and I missed it. Nice catch!

**Pull request merged**